cranelift/fuzzgen/src/function_generator.rs

```diff
@@ -1458,10 +1460,14 @@ where
         is_atomic: bool,
     ) -> Result<(Value, MemFlags, Offset32)> {
         // Should we generate an aligned address
-        let is_aarch64 = matches!(self.target_triple.architecture, Architecture::Aarch64(_));
-        let aligned = if is_atomic && is_aarch64 {
-            // AArch64 has issues with unaligned atomics.
-            // https://github.com/bytecodealliance/wasmtime/issues/5483
+        let supports_unaligned_atomics = !matches!(
+            self.target_triple.architecture,
+            Architecture::Aarch64(_) | Architecture::Riscv64(_)
+        );
+        let aligned = if is_atomic && !supports_unaligned_atomics {
+            // Some backends have issues with unaligned atomics.
+            // AArch64: https://github.com/bytecodealliance/wasmtime/issues/5483
+            // RISCV: https://github.com/bytecodealliance/wasmtime/issues/5882
```

Comment:
I think I'd prefer to write this like so, phrased as "requires aligned" instead of "supports unaligned" and with the comment placed earlier:
```suggestion
        // Some backends have issues with unaligned atomics.
        // AArch64: https://github.com/bytecodealliance/wasmtime/issues/5483
        // RISCV: https://github.com/bytecodealliance/wasmtime/issues/5882
        let requires_aligned_atomics = matches!(
            self.target_triple.architecture,
            Architecture::Aarch64(_) | Architecture::Riscv64(_)
        );
        let aligned = if is_atomic && requires_aligned_atomics {
```

Reply:
Yeah that looks better, thanks!